setup.py

```diff
@@ -8,7 +8,7 @@
     version=__version__,  # noqa:F821
     packages=['rosdep2', 'rosdep2.platforms'],
     package_dir={'': 'src'},
-    install_requires=['catkin_pkg >= 0.4.0', 'rospkg >= 1.0.37', 'rosdistro >= 0.7.0', 'PyYAML >= 3.1'],
+    install_requires=['catkin_pkg >= 0.4.0', 'rospkg >= 1.1.8', 'rosdistro >= 0.7.0', 'PyYAML >= 3.1'],
```

Comment:
The version number needs to also be updated in these two lines: https://github.com/ros-infrastructure/rosdep/blob/fe63aaac265a2ef3823726250815c0d64e9e9b5c/stdeb.cfg#L2-L3